sessionctx/variable/tidb_vars.go

```diff
@@ -886,8 +886,8 @@ const (
 	// TiDBMaxAutoAnalyzeTime is the max time that auto analyze can run. If auto analyze runs longer than the value, it
 	// will be killed. 0 indicates that there is no time limit.
 	TiDBMaxAutoAnalyzeTime = "tidb_max_auto_analyze_time"
-	// TiDBDDLEnableDistributeReorg indicates whether to enable the new Reorg framework.
-	TiDBDDLEnableDistributeReorg = "tidb_ddl_distribute_reorg"
+	// TiDBEnableDistTask indicates whether to enable the distributed execute background tasks(For example DDL, Import etc).
+	TiDBEnableDistTask = "tidb_enable_dist_task"
```

Comment:
We should not change this directly because the old variable name has been released in v6.6.0. Please deprecate it first and then remove it in the next several versions.

Reply:
I will also change the doc, in 7.1.